src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
use for loop and create variable for Lottery before for loop, so you won't need to create a new instance for every ball